vizro-core/src/vizro/tables/_dash_ag_grid.py

```diff
@@ -50,7 +50,11 @@
 def dash_ag_grid(data_frame: pd.DataFrame, **kwargs) -> dag.AgGrid:
     """Implementation of `dash_ag_grid.AgGrid` with sensible defaults to be used in [`AgGrid`][vizro.models.AgGrid].
 
-    Examples
+    Args:
+        data_frame: DataFrame containing the data to be displayed.
+        kwargs: Additional keyword arguments to be passed to the `dash_ag_grid.AgGrid` component.
+
```

Comment:
Shall we add a  Returns as well? To be consistent with e.g. the kpi_card functions and their docstrings. Same for dash_data_table.

```
Returns:
        A Dash Bootstrap Components card (`dbc.Card`) containing the formatted KPI value and reference.

```